app/services/algolia/vacancy_filters_builder.rb

```diff
@@ -14,6 +20,7 @@ def filter_query
     build_filters
 
     filter_array = []
+    filter_array << "(#{@search_filter})"
```

Comment:
The name `@search_filter` is maybe a little opaque in the context of these other filters. Could rename to `@published_filter`.